### DISCUSSION TITLE
Better range: CC2538+CC2592 or CC2652R

### DISCUSSION THREAD

**Discussion by ai-xyz** [Koenkk/zigbee2mqtt#5819] [General]
Hi,

currently i am using a CC2531 with external antenna and i am still quite happy with that stick. But i want to do an update to a more sophisticated stick like zzh! / slaesh's (CC2652R) or a CC2538+CC2592 (from Tindie for example).

The "Supported adapters" section recommends to use either zzh! or slaesh's stick. But somehow the CC2538+CC2592 reads even more powerful due to the CC2592 power amplifier. Also the Z-Stack FW reads that the CC2538+CC2592 allows more direct children and routes. So i can't find why are the CC2652R sticks recommended? What is their benefit compared to the CC2538+CC2592?

I hope i don't missed a conversation where this is already discussed.

**Comment by Koenkk** [comment]
The CC2538 + CC2592 might indeed have a bit more range due to the PA. The CC2538 is however a legacy platform and the SDK is not actively maintained by Texas Instruments anymore (only CC2652/CC1352 sdks). Therefore the CC2652 is recommended. If you really want a PA you can go for a CC1352P board and add an antenna yourself. However the recommended CC2652 sticks should be good for most use cases.
